node/rpc-client/src/client.rs

```diff
@@ -206,4 +217,45 @@ pub mod filecoin_rpc {
     pub async fn chain_read_obj(cid: ChainReadObjParams) -> Result<ChainReadObjResult, Error> {
         call_params(CHAIN_READ_OBJ, cid).await
     }
+
+    /// Wallet
+    pub async fn wallet_new(signature_type: WalletNewParams) -> Result<String, Error> {
```

Comment:
Try to make Result types match up with those in the `rpc_api` in addition to Params. Each one needs to follow the pattern.

Reply:
Yup, skipped over these. Think they were from before they existed? Idk. Fixed them up